### DISCUSSION TITLE
AOSC OS Core 10 Codename Nomination

### DISCUSSION THREAD

**Discussion by MingcongBai** [AOSC-Dev/aosc-os-abbs#4065] [General]
**Welcome to the AOSC OS Core 10 “J” codename nomination!**

For our AOSC OS Core 10 release series, we are asking for nominations of codenames starting with the letter “J.” We stipulate the following rules for your nomination(s):

1. Your codename(s) must start with the Latin letter “J,” but could be in any Latin-alphabet language.
2. You may nominate as many codenames as you’d like. In case of duplicate nominations, the duplicates will be named under the first nominator.
3. Your submission must not contain trademarks or expressions of political, racial, or religious terms, expressions and opinions.
4. Additionally, you are asked to refrain from nominating codenames with references to Mingcong Bai – for instance, the words and derivatives of “Jeff” and “Jelly.”

**Note: The rules above are subject to interpretation by our community contributors. After the nomination deadline, community contributors will convene to audit and filter all nominations based on rule compliance.**

**Nomination will close at 0:00 on July 19th, UTC time, we will publish voting rules and threads in the meantime.**

Thank you again for your submission!

--------

**欢迎参与 AOSC OS Core 10 “J” 代号提名！**

AOSC OS Core 10 的代号将以字母“J”开头，现征集提名。本次代号提名要求及规则如下：

1. 代号应以拉丁字母“J”开头，但可使用拉丁字母表的任意语言。
2. 您可以提名任意个代号，如有重复提名，则以第一位提名者为准。
3. 提名不得包含商标及政治、种族、宗教等内容的名词、概念及观点性内容。
4. 请勿提名任何与社区贡献者白铭骢有关的代号：如关键字“Jeff”、“Jelly”及其其他变体。

**注：如上规则解释权归社区贡献者所有。在征集截止后，社区贡献者将召开例会，届时我们将根据合规性考虑审核及筛选提名。**

**提名截止时间：UTC 时间 7 月 19 日零时。**

感谢您的提名！

**Comment by RedL0tus** [comment]
Jinkela 金坷拉！

**Comment by cthbleachbit** [comment]
* Just-in-Time
* Jordan    -  As in https://en.wikipedia.org/wiki/Jordan_matrix

**Comment by RedL0tus** [comment]
judiciousness
 - n. good judgement.
 - n. the trait of forming opinions by distinguishing and evaluating.

J 开头最长的词之一 / Longest word starting with J

**Comment by OriginCode** [comment]
- Java - Java ([Indonesian](https://en.wikipedia.org/wiki/Indonesian_language): Jawa, Indonesian pronunciation: [[ˈdʒawa]](https://en.wikipedia.org/wiki/Help:IPA/Malay); [Javanese](https://en.wikipedia.org/wiki/Javanese_language): ꦗꦮ; [Sundanese](https://en.wikipedia.org/wiki/Sundanese_language): ᮏᮝ) is one of the [Greater Sunda Islands](https://en.wikipedia.org/wiki/Greater_Sunda_Islands) in [Indonesia](https://en.wikipedia.org/wiki/Indonesia).
- Jingle Bells
- Jet
- Jojo

**Comment by CoelacanthusHex** [comment]
Jacobi - One of the greatest mathematicians in human history, made significant contributions to calculus and elliptic functions.

**Comment by rickliu2000** [comment]
Janesville

**Comment by rickliu2000** [comment]
Jalapeño

**Comment by liushuyu** [comment]
JSON

**Comment by MingcongBai** [comment]
- Jackass - LOL
- Jasmine Rice - Hard not to like!
- Jaywalker - Why follow the rules?
- Joule - Well, that's what compilation does to you

**Comment by Cyanoxygen** [comment]
- Jupiter (4th plant in the Solar System)
- Jitterbug (A form of swing dance) - Not a type of bug! _Well you can make a bug art out of this if you wish._
- [Jaffa cakes](https://en.wikipedia.org/wiki/Jaffa_Cakes) - A cake made by a UK firm, note that this name is not registered. And it is a snack.
- Jackpot - Jackpot!

**Comment by 67au** [comment]
- Jailbreak - But **Hotfix** in previous version

**Comment by Rongronggg9** [comment]
Jam
Junior
Jr
Julian Calendar

**Comment by startmenu** [comment]
Jvav
“我会Jvav啊”

**Comment by LynxCatTheThird** [comment]
Joyan：克里奥尔语“合作”（Jɔyn an togɛda）的部分字母

**Comment by LynxCatTheThird** [comment]
Janes：来自爱沙尼亚语“兔子”（Jänes），吉祥物安安是兔子

**Comment by LynxCatTheThird** [comment]
Jakamyu：来自卢干达语“兔子”（akamyu）

**Comment by LynxCatTheThird** [comment]
Jarnar（或Jarna）：来自马来语“兔子”（arnab）

**Comment by LynxCatTheThird** [comment]
Jakot：来自马其顿语“兔子”（зајакот）

**Comment by CattenLinger** [comment]
JamJar - 蜜罐（大霧
Java 
Jam
Jar

**Reply by MingcongBai** [reply]
"Java" is redundant, nominated by @OriginCode .

**Reply by MingcongBai** [reply]
"Jam" is also redundant, nominated by @Rongronggg9 .

**Comment by MingcongBai** [comment]
John Marston

**Comment by LynxCatTheThird** [comment]
Jiru：来自奥罗莫语“同学”（hiriyaa daree keessa jiru）

**Comment by LynxCatTheThird** [comment]
Jasnic：来自白俄罗斯语“同学”（аднакласнік）

**Comment by shanoaice** [comment]
Javelin: 标枪，皇家海军 J 级驱逐舰之一

**Comment by MariaBombardier** [comment]
JDK

**Comment by SuperMarioSF** [comment]
"Java Editon"

AOSC OS 10 Core "Java Edition"

Also suggesting logo as a cup of coffee but not a hot one (to avoid trademark infringement).

// sounds like something Minecraft related...

**Reply by SuperMarioSF** [reply]
This suggestion only meant to be entertainment purpose.
When this suggestion is actually meant to be used, consider combine idea from @startmenu , becoming: "Jvav Edition".

**Comment by WhiredPlanck** [comment]
JavaScript
or just "JS"

// LOL

**Comment by WhiredPlanck** [comment]
Jarvis（贾维斯）
J.A.R.V.I.S

**Comment by StephDC** [comment]
Juan - Maybe something that does not sound like an English J works. Also works as a pun to some popular Chinese word

> Spanish ver of John or Chinese ver of roll up, all up to your mental status.

**Comment by RedL0tus** [comment]
Jurassic (Version) / 侏罗纪

Dat's an ancient distro isn't it

**Comment by noirgif** [comment]
July version

**Comment by outloudvi** [comment]
* Junna (⭐)
* Journey (hasn't ended yet!)
* Journal (no `ctl`)
* Jingle (惊了)

**Comment by Rongronggg9** [comment]
Jerry - a common English name

**Comment by Rongronggg9** [comment]
Jumbo
J
JJ
JJJ
JJJJ

[![樱桃（车厘子）J级，JJ级，JJJ级，XL级代表什么意思？划分标准是？ - 鱿鱼丝真好吃的回答 - 知乎](https://user-images.githubusercontent.com/15956627/179449394-e228c95c-5a88-4e6b-a9fd-b126a9a2578e.png)](https://www.zhihu.com/question/27350764/answer/1689036747)

**Comment by MingcongBai** [comment]
Thank you all for your nominations, the call for codename is now closed. We will move on to the voting stage - please stay tuned for voting venue.
